requirements/requirements_test.txt

```diff
@@ -1,4 +1,5 @@
 parameterized
 pytest
 black
-isort[requirements]
\ No newline at end of file
+isort[requirements]
+wrapt
```

Comment:
not sure if wrapt has to be part of requirements_test.txt if it is already in requirements.txt